packages/ubuntu_provision/lib/src/accessibility/accessibility_page.dart

```diff
@@ -13,7 +13,9 @@ class AccessibilityPage extends ConsumerWidget with ProvisioningPage {
   @override
   Future<bool> load(BuildContext context, WidgetRef ref) {
     final model = ref.read(accessibilityModelProvider);
-    return model.init().then((_) => true);
+    // This will return false when it is not Gnome, since the model wont be able
+    // to initialize with GSettings.
+    return model.init().then((_) => true, onError: (_) => false);
```

Comment:
Did you try this? Errors are caught and logged in `GnomeAccessibilityService`, so `model.init()` should never throw

Reply:
I did not, I pushed it during the meeting 😄

Reply:
Just checked it by using nonexistent dconf paths and it doesn't work. Sorry, I think my `GnomeAccessibilityService` implementation fails a bit too gracefully in that case and just replies with default values. Maybe we could throw a custom exception in the `_tryGet`/`_trySet` methods

Reply:
What about [this](https://github.com/canonical/ubuntu-desktop-provision/pull/406/commits/67a91f6c18988159f7a240e5f3aafffd364cf20c), if we don't get any options at all back, then we treat it as non-supported?

Reply:
works for me :+1:

Reply:
Ah, if there are default values I guess that won't work either...

Reply:
Seems to work!

Reply:
Ah, haha, your comment didn't pop up until I commented